src/trigger-circleci-workflows.ts

```diff
@@ -103,6 +104,10 @@ export function circlePipelineEndpoint(repoKey: string): string {
   return `/api/v2/project/github/${repoKey}/pipeline`;
 }
 
+function invert(label: string): string {
+  return label.replace(/^ci\//, 'ci/no-');
```

Comment:
How would this affect labels that aren't prefixed with `ci/`?

Reply:
¯\_(ツ)_/¯ I guess it'll just return the original label; I could modify the logic so that if the label isn't prefixed with `ci/` then it doesn't have an inverse case, or we could go with your previous suggestion and make it `no-ci/*` instead of `ci/no-*` (although doing that would require removing the existing `ci/no-build` on the PyTorch repo, and replacing it with `no-ci/build`)